app/routes/events/components/EventList.css

```diff
@@ -60,3 +61,8 @@
     margin-bottom: 40px;
   }
 }
+
+.filterButtons label {
+  display: flex;
+  align-items: center;
+}
```

Comment:
Please include a line at the end of the file 😅 

Make the change, then amend it to your commit:
`git add <file>`
`git commit --amend --no-fix`

Reply:
Or`git add .` to add all the files 😀

Reply:
fixa det, gjorde det bare i vscode source control

Reply:
Faster anyways that writing Eventlist.css. len(".") < len("Eventlist.css")